apps/experiments/views/experiment.py

```diff
@@ -270,6 +275,12 @@ def form_valid(self, form, file_formset):
     def form_invalid(self, form, file_formset):
         return self.render_to_response(self.get_context_data(form=form, file_formset=file_formset))
 
+    def dispatch(self, request, *args, **kwargs):
+        is_chatbot = kwargs.get("new_chatbot", False)
+        if flag_is_active(request, "flag_chatbots") and not is_chatbot:
```

Comment:
nit: swapping the order will prevent the FF check from happening when `is_chatbot` is `True`
```suggestion
        if not is_chatbot and flag_is_active(request, "flag_chatbots"):
```